packages/@angular/cli/commands/build.ts

```diff
@@ -41,10 +41,18 @@ export const baseBuildCommandOptions: any = [
     type: Number,
     default: pollDefault,
     description: 'enable and define the file watching poll time period (milliseconds)'
+  },
+  {
+    name: 'app',
+    type: String,
+    aliases: ['a'],
+    description: 'Specifies app name to use.'
   }
 ];
 
 export interface BuildTaskOptions extends BuildOptions {
+  watch?: boolean;
+  app?: String;
```

Comment:
Add this to https://github.com/angular/angular-cli/blob/master/packages/@angular/cli/models/build-options.ts instead